File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/03.Conditional-Statements-Advanced/02-exercise/06.operationsBetweenNumbers.js
function operationsBetweenNumbers(input) {

    const n1 = Number(input[0]);
    const n2 = Number(input[1]);
    const operator = input[2];

    let result = 0;
    let evenOdd = 0;

    switch (operator) {
        case "+": result = n1 + n2;
            if (result % 2 === 0) {
                evenOdd = "even";
            } else {
                evenOdd = "odd";
            }
            console.log(`${n1} ${operator} ${n2} = ${result} - ${evenOdd}`);
            break;
        case "-": result = n1 - n2;
            if (result % 2 === 0) {
                evenOdd = "even";
            } else {
                evenOdd = "odd";
            }
            console.log(`${n1} ${operator} ${n2} = ${result} - ${evenOdd}`);
            break;
        case "*": result = n1 * n2;
            if (result % 2 === 0) {
                evenOdd = "even";
            } else {
                evenOdd = "odd";
            }
            console.log(`${n1} ${operator} ${n2} = ${result} - ${evenOdd}`);
            break;
        case "/": result = n1 / n2;
            if (n2 === 0) {
                console.log(`Cannot divide ${n1} by zero`);
            } else {
                console.log(`${n1} / ${n2} = ${result.toFixed(2)}`);
            }
            break;
        case "%": result = n1 % n2;
            if (n2 === 0) {
                console.log(`Cannot divide ${n1} by zero`);
            } else {
                console.log(`${n1} % ${n2} = ${result}`);
            }
            break;
    }
}
operationsBetweenNumbers(["11", "8", "*"]);

File: Python_Basics/Homeworks-And-Labs/03.conditional_statements_advanced/02.exercise/06.operations_between_numbers.py
n1 = int(input())
n2 = int(input())
operator = input()

result = 0
evenOdd = 0

if operator == '+':
    result = n1 + n2
    if result % 2 == 0:
        evenOdd = "even"
    else:
        evenOdd = "odd"        
    print(f'{n1} {operator} {n2} = {result} - {evenOdd}')
elif operator == '-':
    result = n1 - n2
    if result % 2 == 0:
        evenOdd = "even"
    else:
        evenOdd = "odd"        
    print(f'{n1} {operator} {n2} = {result} - {evenOdd}')
elif operator == '*':
    result = n1 * n2
    if result % 2 == 0:
        evenOdd = "even"
    else:
        evenOdd = "odd"        
    print(f'{n1} {operator} {n2} = {result} - {evenOdd}')
elif operator == '/':
    if n2 == 0:
        print(f'Cannot divide {n1} by zero')
    else:
        result = n1 / n2
        print(f'{n1} / {n2} = {result:.2f}')
elif operator == '%':
    if n2 == 0:
        print(f'Cannot divide {n1} by zero')
    else:
        result = n1 % n2
        print(f'{n1} % {n2} = {result}')

